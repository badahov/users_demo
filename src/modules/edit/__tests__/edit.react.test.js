'use strict';

import React from 'react';
import EditMain from '../main';

import { configure, mount, shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });

describe('Edit', () => {
    it('Layout Index correctly', () => {
        const tree = renderer.create(<EditMain />).toJSON();

        expect(tree).toMatchSnapshot();
    });
});
