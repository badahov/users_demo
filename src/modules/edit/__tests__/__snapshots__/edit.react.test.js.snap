// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Edit Layout Index correctly 1`] = `null`;
